lib/engine/game/base.rb

```diff
@@ -1009,6 +1010,17 @@ def after_par(corporation)
         end
       end
 
+      def log_later(&block)
```

Comment:
looks like this only works if called once with a matching log_now, could we make it more robust?

also no need to pass in a block, you can just use yield

Reply:
> looks like this only works if called once with a matching log_now, could we make it more robust?

9bc9ba3a has the game return the lines without storing them in `@log_later`; this still only works if something logs the lines later, but in the caller's code it's more obvious now that the caller should do something with the lines. This is also closer to the `log_later` variable that is used in `step/buy_company.rb`

>  no need to pass in a block, you can just use yield

452744bc

Reply:
i think i prefered the previous version maybe rename it

queue_log

flush_log

??

Reply:
Renamed and rebased